packages/device-id/package.json

```diff
@@ -23,14 +23,19 @@
   ],
   "license": "Apache-2.0",
   "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
   "exports": {
     ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/.esm-wrapper.mjs",
-      "types": "./dist/index.d.ts"
+      "require": {
+        "default": "./dist/index.js",
+        "types": "./dist/index.d.ts"
```

Comment:
I don't think anything that's going to use exports.require is actually going to pick up this types field because the default / "node" moduleResolution in typescript was clearly happily using "types" from the root and now with node16 it will use import.types. plus node/js (outside of typescript) is never going to import the types anyway but it is here for consistency.

Reply:
> A "types" condition like this is redundant, since TSC will automatically infer it from the .js file.

is this the case? I have had issues with packages which don't explicitly define it before

Reply:
Chatted with @lerouxb offline, I'd be happy to share an example for anyone interested.